lib/routes.json

```diff
@@ -1885,6 +1885,7 @@
                 "$number": null,
                 "body": {
                     "type": "Array",
+                    "sendValueAsBody": true,
```

Comment:
Yeah it works. Both objects and arrays can be encoded as JSON. It's the format GitHub expects it in. 

> On 14 Apr 2016, at 01:55, Adrien Lemaire <notifications@github.com> wrote:
> 
> In lib/routes.json:
> 
> > @@ -1885,6 +1885,7 @@
> >                  "$number": null,
> >                  "body": {
> >                      "type": "Array",
> > +                    "sendValueAsBody": true,
> hmm does this really work ? Your condition expects a format json, but this body is a Array.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>